samples/integration-tests/build.gradle.kts

```diff
@@ -99,9 +100,4 @@ tasks.withType<Test>().configureEach {
   maxParallelForks = Runtime.getRuntime().availableProcessors() * 2
 }
 
-// See https://github.com/GradleUp/shadow/issues/1540
-configurations.configureEach {
-  if (name.startsWith("kotlinCompilerPluginClasspath")) {
-    attributes { attribute(Bundling.BUNDLING_ATTRIBUTE, objects.named(Bundling.SHADOWED)) }
-  }
-}
+tasks.withType<KotlinJsTest>().configureEach { enabled = false }
```

Comment:
That is unrelated but fixed `:integration-tests:build` (see https://github.com/ZacSweers/metro/actions/runs/16689036869/job/47243849606)

I _think_ there were no JS tests in that module but feel free to revert if needed.